### ISSUE TITLE
[Task]: Create a Custom Domain for Simpler Grants Wiki

### ISSUE THREAD

**Issue by btabaska** [HHS/simpler-grants-gov#1655]
### Summary

Implement a custom domain setup for our public wiki, transitioning from its current static GitBook domain to wiki.simpler.grants.gov. 

This change aims to enhance brand consistency, improve user trust, and provide a more professional web presence. The process involves DNS configuration, SSL certificate setup for HTTPS, and ensuring seamless access to the wiki content under the new domain without affecting the current users' access and experience.

### Acceptance criteria

Domain Configuration: The domain wiki.simpler.grants.gov should be correctly configured to point to the IP address of the current wiki hosting server. Verify the DNS settings propagate correctly across the internet, ensuring the domain directs to the correct server globally.

Redirect Setup: Configure automatic redirects from the old wiki domain/URLs to the new custom domain, preserving the path and query parameters. This ensures that users and search engines are directed to the correct content under the new domain.

**Comment by btabaska** [comment]
Blocked: Currently working with GitBook user support, as well as Terrance and other HHS staff to uncover why the domain redirect is not working. It is currently giving a Cloudflare DNS error that we are working to unravel.

We have determined that HHS does not use cloudflare on any of our domains and does not have an account with cloudflare. This complicates the blocker because we do not have the ability to contact their support.

We have used a CF tool https://cf.sjr.org.uk/tools/check?e5e2335323da40ee85e2bedcb324ec34 to determine that the CNAME 949b098972-hosting.gitbook.io from GitBook seems to be under Cloudflare protection, and have reached back to GitBook for clarification.
